### PR TITLE
13.0 mig project description

### PR DESCRIPTION
**Pull request by emagdalenaC2i**

### CODE REVIEW

**Comment by emagdalenaC2i** [comment]
@astirpe @Saran440 @schout-it and @nikul-serpentcs could you review this PR?

**Review by schout-it — APPROVED** [review]
lgtm

**Comment by sbidoul** [comment]
@emagdalenaC2i please rebase and run `pre-commit run -a` otherwise, ocabot merge will fail.

File: project_description/views/project_view.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        </field>
+    </record>
+
+    <record id="project_search" model="ir.ui.view">
+        <field name="name">Project description</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.view_project_project_filter"/>
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="description"/>
```
Comment (astirpe) [inline project_description/views/project_view.xml:26]: I cannot see this filter in the interface. How can I reach it?

**Review by astirpe — CHANGES_REQUESTED** [review]

File: project_description/views/project_view.xml (hunk 1 above)
Comment (emagdalenaC2i) [reply project_description/views/project_view.xml:26]: @astirpe In Projects, go to  Configuration --> Projects.
Set a custom filter and select Description

**Review by astirpe — APPROVED** [review]
Code review OK. Tested in Runbot.

**Pull request merged**